server/api/election_settings.py

```diff
@@ -46,6 +47,7 @@ def serialize_election_settings(election: Election) -> JSONDict:
         "riskLimit": election.risk_limit,
         "state": election.state,
         "auditType": election.audit_type,
+        "ballotPollingType": election.ballot_polling_type,
```

Comment:
i think this field should only be present for ballot polling audits

Reply:
Ooh, good nudge. I anticipate that we will eventually be supporting a wide variety of math types, not just for ballot polling. E.g., if we get SHANGRLA implemented it will apply to all three methods. So I could make this more general (audit_math_type or something). Thoughts?